st2common/st2common/models/api/datastore.py

```diff
@@ -25,23 +30,49 @@ class KeyValuePairAPI(BaseAPI):
             'id': {
                 'type': 'string'
             },
+            'name': {
+                'type': 'string'
+            },
             'description': {
                 'type': 'string'
             },
-            'name': {
+            'value': {
                 'type': 'string',
                 'required': True
             },
-            'value': {
+            'expire_timestamp': {
```

Comment:
Sensors which use datastore to persist temporary data might want to set TTL so keys which might not be used again later on get cleaned up.